PurrSoft/PurrSoft.Application/Queries/AnimalFosterMapQueries/AnimalFosterMapQueries.cs

```diff
@@ -0,0 +1,33 @@
+﻿using PurrSoft.Application.Common;
+using PurrSoft.Application.Models;
+using PurrSoft.Application.QueryOverviews;
+
+namespace PurrSoft.Application.Queries.AnimalFosterMapQueries;
+
```

Comment:
Are the first two getters necessary if we have GetFilteredAnimalFosterMapsQueries with the filter fields AnimalId and FosterId?

Reply:
No, I initially used them but then forgot to delete them.